downstream/assemblies/platform/assembly-installing-hub-operator-external-db.adoc

```diff
@@ -23,35 +23,51 @@ You can use these instructions to install the {HubName} operator on {OCP} with a
 
 * You have installed the catalog containing the {PlatformName} operators
 
-.Installing the {HubName} operator
+== Installing the {HubName} operator
 
 . Log in to {OCP}.
 . Navigate to *Operators* -> *Operator Hub*.
 . Locate the {PlatformName} catalog and click on it.
 . Click *Install*
-
++
 .Instantiating an instance of {HubName}
-
 [NOTE]
 ====
 {OCP} clusters running on AWS do not support ReadWriteMany without adding NFS or other storage.
 ====
-
 . Navigate to *Operators* -> *Installed Operators*.
-. Locate the {HubName} operator and click on it.
-. Click *Create Instance*.
-. Use either the *Form View* or *YAML view* to configure the instance.
-.. Click *Advanced Configuration* to enter additional resource information.
+. Locate the {HubName} operator and click *Create AutomationHub*.
+
+.Optional - Configure your {HubName} operator route options
+. Click *Advanced configuration*.
+. Under “Ingress type”, click the dropdown menu and select *Route*.
+. Under “Route DNS host”, enter a common host name that the route answers to.
+. Under “Route TLS termination mechanism”, click the dropdown menu and select *Edge* or *Passthrough*.
+. Under “Route TLS credential secret”, click the dropdown menu and select a secret from the list.
+
+
+.Optional - Configure the Ingress type for your {HubName} operator
+. Click *Advanced Configuration*.
+. Under “Ingress type”, click the dropdown menu and select *Ingress*.
```

Comment:
Should the GUI text be bold  - *Ingress type* ? 
https://redhat-documentation.github.io/asciidoc-markup-conventions/ - see "GUI Text"

Not sure if it would be accurate to say something like 
In the *Ingress Type* menu, select *Ingress* 
In the *Ingress annotations* field, enter any annotations to add to the ingress.

Reply:
You're right, it should be bold. Says the same thing on page 208 on the IBM style guide.

I've also changed dropdown to drop-down, as I found from glancing at the style guide